Adding in images to be used for future commit and clarying duration guidelines (#16)
AniList/Submission-Manual: docs/submissions.md

For works such as one-shots, standalone movies, and anthologies, both the start and end date should be set to the same day. This gives it a '_Release Date_' label in the main entry.
:::

::: warning

> At times an anime will be announced using a >24 hour clock, such as saying the anime will start airing on July 3rd at 25:15. This typically will mean it is actually airing on July 4th at 1:15 AM, however this can also sometimes mean at July 3rd at 1:15 AM. Conversely they will announce an anime is airing on July 4th at 1:15 AM and that can sometimes mean it will air on July 3rd at 1:15 AM.

Reviewer: ```suggestion
Occasionally, an anime will be announced using a >24 hour clock, such as saying the anime will start airing on July 3rd at 25:15. This typically will mean it is actually airing on July 4th at 1:15 AM, however this can also sometimes mean at July 3rd at 1:15 AM. Conversely they will announce an anime is airing on July 4th at 1:15 AM and that can sometimes mean it will air on July 3rd at 1:15 AM.
```

Please use the average length \(_**mode**_\) of episodes, rounded to the nearest minute. The 23-25 mark is standard for most TV anime. This can be left blank until a full-length episode has been released.

::: tip

> The mean duration can be used for entries with wildly varying lengths, such as [Genius Party](https://anilist.co/anime/3508/GeniusParty). Another way to view this is that if a standard episode length can not accurately be stated for a work, that the work should use the mean for the duration. For an example of such, consider [Princess Tutu](https://anilist.co/anime/721/Princess-Tutu/). Low episode count works that have any variance in episode length will typically use the mean.

Reviewer: ```suggestion
The mean duration can be used for entries with wildly varying lengths, such as [Genius Party](https://anilist.co/anime/3508/GeniusParty). Another way to view this is that if a standard episode length can not accurately be stated for a work, that the work should use the mean for the duration. For an example of such, consider [Princess Tutu](https://anilist.co/anime/721/Princess-Tutu/). Works with low episode counts with episodes that vary in length will typically use the mean episode length.
```

::: tip
If you need to fix or generate a new schedule for a longer running anime, such as [Meitantei Conan](https://anilist.co/anime/235/Meitantei-Conan/) you will need to adjust the episode # to start at appropriately. The starting episode date should also be modified to reflect the airing date of the "start generating at episode #" episode you had selected.
:::

::: tip
If an anime releases multiple episodes at one time, in order to correct the "next airing episode" you can click on past dates in the calender which will bump up the next airing episode number accordingly.
:::

::: warning
At times an anime will be announced using a >24 hour clock, such as saying the anime will start airing on July 3rd at 25:15. This typically will mean it is actually airing on July 4th at 1:15 AM, however this can also sometimes mean at July 3rd at 1:15 AM. Conversely they will announce an anime is airing on July 4th at 1:15 AM and that can sometimes mean it will air on July 3rd at 1:15 AM.
:::

> ![Schedule Generator panel](./img/Schedule_Generator.png)

Reviewer: It looks like this may need to be lowercase:

```suggestion
![Schedule Generator panel](./img/schedule_generator.png)
```

### Next Airing Episode

The next airing episode area displays when the next episode is set to air according to the schedule and what episode # that will be. Please always check this before submitting a change to the schedule.

### Preview

> The preview area is similar to next airing episode area, however it allows a moderator to check to ensure they set up the airing dates correctly.

Reviewer: ```suggestion
The preview area is similar to next airing episode area, however it allows a moderator to check that airing dates have been set up correctly.
```
